app/src/main/java/org/schabi/newpipe/util/FilePickerActivityHelper.java

```diff
@@ -0,0 +1,18 @@
+package org.schabi.newpipe.util;
+
+import android.os.Bundle;
+import org.schabi.newpipe.R;
+
+public class FilePickerActivityHelper extends com.nononsenseapps.filepicker.FilePickerActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        String currentTheme = ThemeHelper.getSelectedTheme(this);
+        if(currentTheme.equals("light_theme")) {
```

Comment:
You can use `ThemeHelper#isLightThemeSelected` instead of hard coding it.

Reply:
Good catch. I fixed it.